sentry/src/main/java/io/sentry/HubAdapter.java

```diff
@@ -154,23 +154,23 @@ public IHub clone() {
   }
 
   @Override
-  public SentryId captureTransaction(SentryTransaction transaction, Object hint) {
+  public SentryId captureTransaction(ITransaction transaction, Object hint) {
     return Sentry.getCurrentHub().captureTransaction(transaction, hint);
   }
 
   @Override
-  public SentryTransaction startTransaction(TransactionContext transactionContexts) {
+  public ITransaction startTransaction(TransactionContext transactionContexts) {
```

Comment:
the suggestion button is just gone oO
I'd add `@NotNull` to the return type

Reply:
👍